src/main/java/com/fasterxml/jackson/databind/ser/std/MapSerializer.java

```diff
@@ -182,14 +191,32 @@ protected MapSerializer(Set<String> ignoredEntries,
         _suppressNulls = false;
     }
 
+    /**
+     * @since 2.5
```

Comment:
Add `@deprecated Since 2.12` to make it easier to track, remove from 3.0 (same for all other compatibility methods)

Reply:
OK I'll add `@Deprecated` as well if that's okay